knowledge-base/pass-parameters-to-html5-report-viewer-from-a-views-model.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Pass parameters to the HTML5 Report Viewer from the Model of a View
+description: "Learn the specifics of passing a parameter to the HTML5 Report Viewer from the Model of a View"
+type: how-to
+page_title: Pass parameters to the HTML5 Report Viewer from the Model of a View
+slug: pass-parameters-to-html5-report-viewer-from-a-views-model
+position: 
+tags: 
+ticketid: 1587587
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Project Type</td>
+			<td>ASP.NET Core</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+When working with the HTML Report Viewer jQuery widget, passing parameters to it from the model of the view can be a bit tricky sometimes.

```

Comment:
Try to be a bit more specific here, maybe mention that this is not as simple as expected due to how the model is serialized?